Source/WebCore/page/scrolling/mac/ScrollerMac.h

```diff
@@ -55,7 +55,10 @@ class ScrollerMac {
     CALayer *hostLayer() const { return m_hostLayer.get(); }
     void setHostLayer(CALayer *);
 
+    RetainPtr<NSScrollerImp> takeScrollerImp() { return WTFMove(m_scrollerImp); }
```

Comment:
Better to use `std::exchange(m_scrollerImp, { })`;

Reply:
Fixed.